modules/mod_base/templates/security.txt.tpl

```diff
@@ -0,0 +1,24 @@
+{% with m.site.security as security %}
+# Our security address
+{% for k, c in security.contact %}
+Contact: {% if k == 'email' %}mailto:{% endif %}{{ c }}
+{% empty %}
+# Please contact the site administrator as no contact details are configured for security issues specifically.
+{% endfor %}
```

Comment:
Should it be possible to have multiple contact addresses configured?
Is that why it has the for..in..?

Reply:
Yes, the model supports an email and url.